validator/tests/test_events/tests.py

```diff
@@ -393,6 +394,41 @@ def test_broadcast_events(self):
             validator_pb2.Message.CLIENT_EVENTS,
             event_list, connection_id="test_conn_id", one_way=True)
 
+    def test_catchup_subscriber(self):
+        """Test that catch subscriber handles the case of:
+        - no blocks (i.e. the genesis block has not been produced or received
```

Comment:
end paren

Reply:
Probably not going to correct that here, but in master